.github/workflows/branch_sync.yml

```
@@ -17,4 +17,5 @@ jobs:
     uses: cylc/release-actions/.github/workflows/branch-sync.yml@v1
     with:
       head_branch: ${{ inputs.head_branch }}
+      ignore_branches: 2019.01.x
```

In case we end up with another 2019 branch for whatever reason
```suggestion
      ignore_branches: '2019.*'
```